### PR TITLE
Feat/v3 rtss

### PR DESCRIPTION
**Pull request by dxlin**
### PR Checklist

- [x] Brief description of changes
- [ ] Links to any relevant issues
- [ ] Required status checks are passing
- [ ] User cases if changes impact the user's experience
- [x] `@mention` a maintainer to request a review

Hi @sedghi, sorry for the delay with this. 

This is the code that exports/converts current segmentations to RTSS dicom files. This is invoked via the commands exportRTSS or uploadRTSS (depending if want to download to local or upload to PACS).

Originally we had discussed putting this code within cornerstone or dcmjs, but ultimately I felt like it may have been easier and more concise to keep this as an extension. At the moment the RTSS upload portion only uploads a new RTSS dicom, but once I can see the import code I can also work to revise it so that it replaces if the segmentation affected was from a previously existing RTSS dicom.

Sorry for the delay, had some difficulties with bugs the conversion when segmentations intersect with the border of the image volume.

[blog]: https://circleci.com/blog/triggering-trusted-ci-jobs-on-untrusted-forks/
[script]: https://github.com/jklukas/git-push-fork-to-upstream-branch

### CODE REVIEW

File: platform/viewer/pluginConfig.json [hunk 1]
```diff
@@ -31,6 +31,10 @@
     {
       "packageName": "@ohif/extension-cornerstone-dicom-seg",
       "version": "3.0.0"
+    },
+    {
+      "packageName": "dicom-rtss",
```
Comment (sedghi) [inline platform/viewer/pluginConfig.json:36]: can we name this extension dicom-rt?

Comment (sedghi) [reply platform/viewer/pluginConfig.json:36]: @ohif/extension-dicom-rt

**Comment by sedghi** [comment]
For some reason the exported RT looks like this 
image

Do you have any idea why?

File: platform/viewer/pluginConfig.json (hunk 1 above)
Comment (dxlin) [reply platform/viewer/pluginConfig.json:36]: Will do, I will make this modification along with the colors in the next commit.

**Comment by dxlin** [comment]
Would you be able to also show the original segmentation?

It looks like the contour is outside of the volume, but that should not be happening as the vtkjs routine to my knowledge is based on the segmentation volume, which should be identical to the image volume.

**Comment by sedghi** [comment]
Any chance you can look at my comment?

**Comment by dxlin** [comment]
> Any chance you can look at my comment?

Sorry I hadn't had a chance to look at this again for a while, I'll try to take a look later this week. 

Would you be able to note the tool/steps you used to create the segmentation? When I did my testing I had only used the brush tool so I may need to test with a variety of tools.

**Comment by sedghi** [comment]
We should move this logic inside @cornerstonejs/adapters

**Comment by sedghi** [comment]
@dxlin as discussed we are going to close this and you will create a new PR

**Pull request closed without merging**